frontend/screens/FilterScreen.tsx

```diff
@@ -16,8 +16,8 @@ function FilterScreen({ navigation }) {
 
   const [text, onChangeText] = React.useState("");
   const [textLength, setTextLength] = React.useState(MAX_LENGTH);
-
-  const [genre, setGenre] = React.useState(-1);
+  
+  const [genres, setGenres] = React.useState(0); 
```

Comment:
genres is a bit mask that represents which genres are chosen (ie. 0 -> none, 1 -> first genre, 11 -> first and second genre)